src/api.js

```diff
@@ -4,8 +4,14 @@ const boardGamesApi = axios.create({
     baseURL: "https://board-games-api-89zv.onrender.com/api"
 });
 
-export function fetchReviews() {
-    return boardGamesApi.get("/reviews")
+export function fetchReviews(categoryQuery) {
+
+    let fetchString = "/reviews"
+
+    if (categoryQuery) {
+        fetchString += `?category=${categoryQuery}`
+    }
+    return boardGamesApi.get(fetchString)
```

Comment:
This is a perfect use case for Axios params rather than having to manually create the endpoint and query yourself. Take a look in the Axios notes for an example